helm-charts/charts/kube-starrocks/charts/starrocks/templates/init-pwd/job.yaml

```diff
@@ -7,6 +7,14 @@ metadata:
 spec:
   template:
     spec:
+  {{- if .Values.starrocksFESpec.imagePullSecrets }}
+      imagePullSecrets:
+  {{- toYaml .Values.starrocksFESpec.imagePullSecrets | nindent 6 }}
+  {{- end }}
+  {{- if .Values.starrocksFESpec.affinity }}
+      affinity:
+  {{- toYaml .Values.starrocksFESpec.affinity | nindent 8 }}
```

Comment:
Is `nindent 8` correct? and why `{{- toYaml .Values.starrocksFESpec.imagePullSecrets | nindent 6 }}` is 6?